providers/azurecompute-arm/src/main/java/org/jclouds/azurecompute/arm/domain/LoadBalancerSKU.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.azurecompute.arm.domain;
+
+import org.jclouds.azurecompute.arm.util.GetEnumValue;
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.json.SerializedNames;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class LoadBalancerSKU {
+
+   public static enum LoadBalancerSKUName {
+      Basic, Standard, Unrecognized;
+
+      public static LoadBalancerSKUName fromValue(final String text) {
+         return (LoadBalancerSKUName) GetEnumValue.fromValueOrDefault(text, LoadBalancerSKUName.Unrecognized);
+      }
+   }
+
+   @Nullable
```

Comment:
I don't think this is nullable. Can the only field in the object be null?

Reply:
You're right, the whole `LoadBalancerSKU` can be null and will default to `Basic` but this field is mandatory. Will change